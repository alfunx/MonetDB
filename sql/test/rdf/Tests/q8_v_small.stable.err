stderr of test 'q8_v_small` in directory 'sql/test/rdf` itself:


# 15:59:45 >  
# 15:59:45 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=35378" "--set" "mapi_usock=/var/tmp/mtest-16175/.s.monetdb.35378" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_rdf" "--set" "embedded_r=yes"
# 15:59:45 >  

# builtin opt 	gdk_dbpath = /home/niels/scratch/monetdb/Linux-x86_64/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 35378
# cmdline opt 	mapi_usock = /var/tmp/mtest-16175/.s.monetdb.35378
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /home/niels/scratch/monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_rdf
# cmdline opt 	embedded_r = yes
# cmdline opt 	gdk_debug = 536870922

# 15:59:45 >  
# 15:59:45 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-16175" "--port=35378"
# 15:59:45 >  


# 15:59:54 >  
# 15:59:54 >  "Done."
# 15:59:54 >  

