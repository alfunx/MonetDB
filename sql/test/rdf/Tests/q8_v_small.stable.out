stdout of test 'q8_v_small` in directory 'sql/test/rdf` itself:


# 15:59:45 >  
# 15:59:45 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=35378" "--set" "mapi_usock=/var/tmp/mtest-16175/.s.monetdb.35378" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_rdf" "--set" "embedded_r=yes"
# 15:59:45 >  

# MonetDB 5 server v11.22.0
# This is an unreleased version
# Serving database 'mTests_sql_test_rdf', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs and 128bit integers dynamically linked
# Found 7.332 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.nes.nl:35378/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-16175/.s.monetdb.35378
# MonetDB/GIS module loaded
# Start processing logs sql/sql_logs version 52200
# Start reading the write-ahead log 'sql_logs/sql/log.5'
# Finished reading the write-ahead log 'sql_logs/sql/log.5'
# Finished processing logs sql/sql_logs
# MonetDB/SQL module loaded
# MonetDB/R   module loaded

Ready.

# 15:59:45 >  
# 15:59:45 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-16175" "--port=35378"
# 15:59:45 >  

#with A as (
#		(select subj, obj FROM prop14657240_pso where subj = 12854543) 
#		 UNION ALL
#		(select subj, obj FROM prop14660343_pso where subj = 12854543) 
#		 UNION ALL
#		(select subj, obj FROM prop17582582_pso where subj = 12854543) 
#		 UNION ALL
#		(select subj, obj FROM prop14657202_pso where subj = 12854543) 
#		 UNION ALL
#		(select subj, obj FROM prop14657244_pso where subj = 12854543) 
#		 UNION ALL
#		(select subj, obj FROM prop14657245_pso where subj = 12854543) 
#		 UNION ALL
#		(select subj, obj FROM prop14660087_pso where subj = 12854543) 
% .trip # table_name
% subj # name
% int # type
% 1 # length

# 15:59:54 >  
# 15:59:54 >  "Done."
# 15:59:54 >  

